pulse/broker/src/main/java/gov/ca/emsa/pulse/broker/entity/PatientEntity.java

```diff
@@ -43,17 +42,6 @@
 	@Column(name = "gender")
 	private String gender;
 	
-	@Column(name = "phone_number")
-	private String phoneNumber;
-	
-	@Column(name = "address_id")
```

Comment:
If you put the address_id back in the data model it could go back here as well until the address gets changed.